dev/ProgressBar/ProgressBar.xaml

```diff
@@ -0,0 +1,118 @@
+                                            </EasingDoubleKeyFrame>
+                                        </DoubleAnimationUsingKeyFrames>
+                                    </Storyboard>
+                                </VisualState>
+                                <VisualState x:Name="Error">
+                                    <Storyboard>
+                                        <DoubleAnimation
+                                            Storyboard.TargetName="ProgressBarIndicator"
+                                            Storyboard.TargetProperty="Opacity"
+                                            To="0"
+                                            Duration="0:0:0.25" />
+                                        <DoubleAnimationUsingKeyFrames
+                                            Storyboard.TargetName="ProgressBarIndicator"
+                                            Storyboard.TargetProperty="(UIElement.RenderTransform).(CompositeTransform.TranslateX)"
+                                            Duration="0:0:0.75">
+                                            <EasingDoubleKeyFrame KeyTime="0:0:1" Value="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.ContainerAnimationStartPosition}">
+                                                <EasingDoubleKeyFrame.EasingFunction>
+                                                    <CubicEase/>
+                                                </EasingDoubleKeyFrame.EasingFunction>
+                                            </EasingDoubleKeyFrame>
+                                        </DoubleAnimationUsingKeyFrames>
+                                    </Storyboard>
+                                </VisualState>
+                                <VisualState x:Name="Paused">
+                                    <Storyboard>
+                                        <ObjectAnimationUsingKeyFrames
+                                            Storyboard.TargetName="ProgressBarIndicator"
+                                            Storyboard.TargetProperty="Fill" >
+                                            <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource SystemBaseMediumLowColor}" />
+                                        </ObjectAnimationUsingKeyFrames>
+                                        <DoubleAnimation
+                                            Storyboard.TargetName="ProgressBarIndicator"
+                                            Storyboard.TargetProperty="Opacity"
+                                            To="{ThemeResource ProgressBarIndicatorPauseOpacity}"
+                                            Duration="0:0:0.25" />
+                                    </Storyboard>
+                                </VisualState>
+                            </VisualStateGroup>
+                        </VisualStateManager.VisualStateGroups>
+
+                        <Border x:Name="ProgressBarRoot"
```

Comment:
Do we want rounded corners on this @chigy?

Reply:
This is what it would look like if it did go rounded...

![image](https://user-images.githubusercontent.com/7389110/67036427-8cc67600-f113-11e9-86ef-b44f8e7ac14b.png)

Reply:
Will the Updated Indeterminate Style have a gradient on it?

Reply:
> Will the Updated Indeterminate Style have the gradient on it?

I believe the finalised design spec does not use a gradient

Reply:
There's a link to the [visual comp](https://github.com/microsoft/microsoft-ui-xaml-specs/blob/user/chigy/ControlUpdates/active/ControlUpdates/images/Progress.png) in #1169. The comp shows rounded corners. The comp does not show any gradients.

While the comp covers both progress bar and progress ring, this change is only for progress bar, so please ignore the ring-specific visuals for now.

Reply:
would the 1% mark have a minimum width of 2x the corner radius?

So if the corner radius is 2px, the progress bar minimum would be 4x4px (forming a circle)

Reply:
> would the 1% mark have a minimum width of 2x the corner radius?
The minimum width is simply 1px right now. But should it be? @YuliKl

Reply:
Doesn't the 1% width actually dependent on the overall width of the progress bar? I would think that if I request a ProgressBar that's 1000px wide, it shows 10px at 1%. I don't think we should special-case any of the intermediate values. I would leave the code as is.

Reply:
I did say minimum width, and this is for the accent coloured value element.  With the corner radius applied to it, if the width was ever smaller than the corner radii of all four corners, would it not render oddly?

Reply:
Whether or not it renders oddly also depends on the height of the progress bar. 

My recommendation is to avoid touching this code until the control ships, and file bugs in case we observe problems later.

Reply:
<img src="https://user-images.githubusercontent.com/7976322/67345394-befb1d80-f4ef-11e9-9f54-bece7c9bb3ea.png" width="200">

This is what a 1px indicator width currently renders to. I agree, let's wait until the control ships. I personally don't see a problem with it, however.